content/en/data-playground/dc-metro.md

```diff
@@ -5,13 +5,14 @@ subtitle: "Stock Market Data"
 draft: true
 image: img/data-playground/finnhub.png
 summary: Connect to stock market data and start experimenting with financial models and apps.
-githhub_link: "github.com"
+github_link: "github.com"
```

Comment:
Good catch! When the other files are updated, we should check if this property is spelled correctly.